	- try to avoid having to have gtk-doc stuff
	- and try to prevent the mess involved with it
--- libqmi-1.12.4.orig/docs/reference/libqmi-glib/Makefile.am	2015-02-10 17:31:06.000000000 +0100
+++ libqmi-1.12.4/docs/reference/libqmi-glib/Makefile.am	2015-02-20 18:24:30.739430307 +0100
@@ -73,9 +73,9 @@ expand_content_files =
 
 extra_files =
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
 
-EXTRA_DIST += \
+EXTRA_DIST = \
 	libqmi-glib-common.sections \
 	$(DOC_MODULE)-sections.txt \
 	$(DOC_MODULE)-sections.mstamp \
@@ -83,7 +83,7 @@ EXTRA_DIST += \
 	$(DIAGRAMS) \
 	$(NULL)
 
-CLEANFILES += \
+CLEANFILES = \
 	$(DOC_MODULE)-decl-list.txt \
 	$(DOC_MODULE)-decl.txt \
 	$(DOC_MODULE)-overrides.txt \
